src/app/tests/suites/certification/Test_TC_OPSTATE_1_1.yaml

```diff
@@ -32,7 +32,6 @@ tests:
                 value: nodeId
 
     - label: "TH reads the ClusterRevision attribute from the DUT"
-      PICS: OPSTATE.S.Afffd
```

Comment:
Why these PICS removals?

Reply:
The Global Attribute PICS have been blanket removed from everywhere as I believe they're not used.  See all of the cluster TPs and all of the <cluster>_1_1 YAML tests in the repo.  So I'm just following the pattern as instructed.